website/blog/2024-09-19-bootc-microshift.md

```diff
@@ -0,0 +1,300 @@
+
+```sh
+$ ssh redhat@localhost -p 2222
+```
+
+4. Move the secret to `/etc/crio/openshift-pull-secret`:
+
+```sh
+$ sudo mv pull-secret.txt /etc/crio/openshift-pull-secret
+```
+
+5. Restart the `microshift` service:
+
+```sh
+$ sudo systemctl restart microshift
+```
+
+### Listing pods
+
+Below we will SSH into the virtual machine and confirm that MicroShift is deploying Pods correctly:
+
+1. SSH into the VM:
+
+```sh
+$ ssh redhat@localhost -p 2222
+```
+
+2. Copy the generated `kubeconfig` file to `~/.kube/config`:
+
+```sh
+$ mkdir -p ~/.kube
+$ sudo cp /var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
+$ sudo chown redhat ~/.kube/config
+```
+
+3. Verify Pods are running by using `oc` or `kubectl`:
+
+```sh
+$ kubectl get pods -A
+NAMESPACE                  NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system                csi-snapshot-controller-856bb8b9bc-9n7lj   1/1     Running   1          3d23h
+kube-system                csi-snapshot-webhook-7c64d4d4d7-98v6l      1/1     Running   1          3d23h
+openshift-dns              dns-default-n2td4                          2/2     Running   2          3d23h
+openshift-dns              node-resolver-7cslg                        1/1     Running   1          3d23h
+openshift-ingress          router-default-7cbc67954b-nqqc6            1/1     Running   1          3d23h
+openshift-ovn-kubernetes   ovnkube-master-zcqw5                       4/4     Running   5          3d23h
+openshift-ovn-kubernetes   ovnkube-node-crnn9                         1/1     Running   2          3d23h
+openshift-service-ca       service-ca-6799f567-k7lsc                  1/1     Running   1          3d23h
+```
+
+### Using Podman Desktop to verify MicroShift
+
+Alternatively, you can copy the MicroShift configuration file to your local machine and test it remotely on Podman Desktop.
+
+1. On your local machine, create the `.kube` directory if it does not exist already:
+
+```sh
+$ mkdir ~/.kube
+```
+
+2. Copy the remote `kubeconfig` file to a local configuration file:
+
+Within MicroShift, a `kubeconfig` file is automatically created at `/var/lib/microshift/resources/kubeadmin/kubeconfig`.
+
+Copy the file over to your local `.kube/config`
+
+```sh
+scp -P 2222 redhat@localhost:/var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
+```
+
+3. Use Podman Desktop to verify the MicroShift cluster:
+
+Podman Desktop will automatically detect your `.kube/config` file.
+
+Note: You may need to modify your `.kube/config` file to reflect the correct domain or IP address of your cluster.
+
+![cluster](img/bootc-microshift/cluster.png)
+
+### Storage configuration
+
+By default, storage configuration [requires an LVM partition](https://github.com/openshift/microshift/blob/main/docs/contributor/storage/default_csi_plugin.md#default-volume-group) and thus LVMS will not be deployed. An alternative storage solution is required for storage utilization.
```

Comment:
The problem is not in the limitation of MicroShift storage driver. It's because you chose to use raw partition. When booting from DVD for example, it's possible to configure storage using kickstart as described [here](https://github.com/openshift/microshift/blob/main/docs/contributor/image_mode.md?plain=1#L262) for example.